src/components/ProfileForm/components/ProfileFormStep0.tsx

```diff
@@ -0,0 +1,59 @@
+const ProfileFormStep0 = () => {
+  return (
+    <>
+      <form className="m-1 p-1 mb-4">
+        <fieldset className="mb-2">
+          <legend className="text-base mb-2 font-bold">
+            Which gender do you identify as?
+          </legend>
+          <div className="flex flex-col">
+            <label>
+              <input name="gender" value="female" type="radio" />
+              <span className="pl-1"> Female </span>
+            </label>
+            <label>
+              <input type="radio" name="gender" value="male" />
+              <span className="pl-1"> Male </span>
+            </label>
+            <label htmlFor="other">
+              <input id="other" type="radio" name="gender" value="other" />
+              <span className="pl-1"> Other </span>
+            </label>
+          </div>
+        </fieldset>
+        <fieldset className="mb-2">
+          <legend className="mb-2 font-bold">
+            What is your date of birth?
+          </legend>
+          <label>
+            <input name="date_of_birth" value="date_of_birth" type="date" />
+          </label>
```

Comment:
We should have some text here, otherwise I don't know what the date is about.

Reply:
<img width="229" alt="Screenshot 2023-07-05 at 18 38 58" src="https://github.com/damdiana/beauty-app/assets/117006466/5a5160b5-a0c3-4d8c-8199-5819841f65bb">
 There is a mention in the legend that the date requested is the birthdate